azure-specialized-workloads/avd/recommendations.yaml

```diff
@@ -1,154 +1,181 @@
-- description: Use Private link when connecting to File Share or Key Vault
-  aprlGuid: dc55be60-6f8c-461e-a9d5-a3c7686ed94e
+- description: Monitor service health and resource health for AVD
+  aprlGuid: 0bf1a2bb-7617-4ab2-a784-e7ea40c5f01b
   recommendationTypeId: null
-  recommendationControl: Security
-  recommendationImpact: Medium
+  recommendationControl: Governance
+  recommendationImpact: Low
   recommendationResourceType: n/a
   recommendationMetadataState: Active
   longDescription: |
-    Private Link is available for other Azure services that work in conjunction with Azure Virtual Desktop, such as Azure Files and Key Vault. From a resiliency standpoint, we recommending implementing private endpoints for these services to reduce exposure to potential internet-related issues such as latency, packet loss, and/or downtime. This can lead to more reliable communication between AVD and dependent services.
-  potentialBenefits: Enhances AVD reliability
+    Use Service Health to stay informed about the health of the Azure services and regions that you use to insure their availability. Set up Service Health alerts so that you stay aware of service issues, planned maintenance, or other changes that might affect your Azure Virtual Desktop resources. Use Resource Health to monitor your VMs and storage solutions.
+  potentialBenefits: Enhanced AVD error tracking and resolution
   pgVerified: true
   publishedToLearn: false
   publishedToAdvisor: false
-  automationAvailable: no
+  automationAvailable: ARG
   tags: null
   learnMoreLink:
     - name: Learn More
-      url: "https://learn.microsoft.com/en-us/azure/architecture/example-scenario/wvd/windows-virtual-desktop#azure-virtual-desktop-limitations"
-    - name: Private Link
-      url: "https://learn.microsoft.com/en-us/azure/well-architected/azure-virtual-desktop/networking#private-endpoints-private-link"
+      url: "https://learn.microsoft.com/azure/well-architected/azure-virtual-desktop/monitoring#resource-health"
 
-- description: Deploy Domain Controllers and DNS Servers in Azure Virtual Network Across Availability Zones
-  aprlGuid: 99bf5c94-aa68-4bb3-8b7f-45d1c5f09b5d
+- description: Configure AVD Insights workbook
```

Comment:
Confirmed with @danycontre that he intends these to be different recommendations which is why they have new guid